components/datepicker/events.md

```diff
@@ -213,6 +214,56 @@ The `OnBlur` event fires when the component loses focus.
 }
 ````
 
+## OnCalendarCellRender
+
+The `OnCalendarCellRender` event fires when each calendar cell in each view is about to render. It allows you to see which view it is in, what its date, and you can set the Class for the `<td>` element based on your business logic.
+
+The event handler receives as an argument an `DatePickerCalendarCellRenderEventArgs` object that contains:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | Lets you set a custom CSS class to the calendar cell DOM element. |
+| `Date` | `DateTime` | The date of the calendar cell. |
+| `View` | `CalendarView` enum <br /> `Month` | The currently visible view. You can use it to determine if the calendar is rendering the MonthView, YearView, and so on. |
+
+>caption Handle the OnCalendarCellRender event.
+
+````CSHTML
+@* Customize the calendar cells using the OnCalendarCellRender event. *@
+
+<TelerikDatePicker OnCalendarCellRender="@OnCalendarCellRenderHandler"
+                   @bind-Value="datePickerValue"
+                   Width="295px">
+</TelerikDatePicker>
+
+@code {
+    DateTime datePickerValue { get; set; } = DateTime.Now;
+
+    private void OnCalendarCellRenderHandler(DatePickerCalendarCellRenderEventArgs args)
+    {
+        if (args.View == CalendarView.Month)
+        {
+            args.Class = args.Date.Day % 3 == 0 ? "special" : "";
+        }
+        else if (args.View == CalendarView.Decade)
+        {
+            args.Class = args.Date.Year == 2020 ? "special" : "";
+        }
+    }
+}
+
+<style>
+    .special {
+        color: white;
+        background-color: greenyellow;
+        font-weight: bold;
+    }
+    /* You can inspect the built-in rendering with the browser dev tools
+        to see how to apply heavier selectors and to also use classes the DatePicker
```

Comment:
There is no such thing as "heavy" selector. There are "more specific selectors". It's not clear why we are suggesting this.

You can also link the "How to override Theme Styles" article.